ingestion/src/metadata/ingestion/source/database/mssql/lineage.py

```diff
@@ -26,10 +26,6 @@
 
 
 class MssqlLineageSource(MssqlQueryParserSource, LineageSource):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.start = self.start.replace(tzinfo=None)
-        self.end = self.end.replace(tzinfo=None)
```

Comment:
`.replace(tzinfo=None)` is handled in `helpers.py` itself so we don't need it here